docs/specs/clients/notify/client-sdk-api.md

```diff
@@ -51,23 +51,12 @@ abstract class Client {
     hasMoreUnread: boolean,
   }>
 
-  // get notification by ID
-  public abstract getNotification(params: {
-    topic: string,
-    id: string,
-  }): Promise<NotifyNotificationRecord>
-
   // mark notification as read
   public abstract markNotificationsAsRead(params: {
     topic: string,
-    ids: string[],
+    notificationIds: string[],
   }): Promise<NotifyNotificationRecord>
```

Comment:
How about we rename it to changeNotificationsReadState and add boolean flag

Reply:
But we won't have the ability to mark as unread yet. Hm. Maybe it can be

```typescript
public abstract changeNotificationReadState(params: {
  topic: string,
  notificationIds: string[]
  read?: true
})
```

Where read is optional and if provided has to be `true`. Down the line it can change to `read: boolean` so it's not a breaking change?

cc @chris13524

Reply:
Makes sense

Reply:
> Down the line it can change to read: boolean so it's not a breaking change?

Changing to make it non-optional is a breaking change too.

Reply:
Aside: We could make the RPC method [name] even more generic, e.g. `updateNotification`.

But for this function name I'm fine with the proposal.

Reply:
wouldn't have to change it to non optional, it would just be `read = true` by default

Reply:
yeah, lets remove optional from above suggestion